assets/telekube/resources/app.yaml

```diff
@@ -138,6 +138,9 @@ nodeProfiles:
           - i2.2xlarge
   - name: master
     description: "Telekube Master"
+    taints:	
```

Comment:
Did you add this temporarily just for testing, or do you wanna keep it in the telekube app?

Reply:
I did add it for testing, however, I left it in the PR since I do think it makes sense to keep, when someone is specifically using a node profile of master, that we do include the taints.